### ISSUE TITLE
Putting a die in a poison?ed card at the start of a turn, before the poison?ed animation, lets you use the card regularly without lowering the countdown

### ISSUE THREAD

**Issue by ncrecc** [TerryCavanagh/diceydungeonsbeta#329]

**Comment by ncrecc** [comment]
Might also apply to silence, shock and shock?

**Comment by lcenine** [comment]
Accidentally did this.

Had a dice ready as round began, wizard had used poison? spell. When the round began I released the dice before the poison? animation started/completed and the equipment (it was a  molotov cocktail) was activated without the poison? countdown penalty.

Not sure how hard it would be to try to exploit.

**Comment by ncrecc** [comment]
This should probably be higher-priority

**Comment by ncrecc** [comment]
![bajump](https://user-images.githubusercontent.com/11216597/62733957-bff2d780-b9f5-11e9-9816-26487981810e.gif)
Also applies to weakened cards (Bump- is max 3)

**Comment by ncrecc** [comment]
![bajurn](https://user-images.githubusercontent.com/11216597/62734399-f846e580-b9f6-11e9-8113-8a56d062e71c.gif)
With amazing timing, you can do the same with burn? and multiuse equipment

**Comment by ncrecc** [comment]
Confirmed for shock, alt shock, silence, and alt poison:
![shock](https://user-images.githubusercontent.com/11216597/62736772-31ce1f80-b9fc-11e9-8fb5-b620d130c2ba.gif)
![altshock](https://user-images.githubusercontent.com/11216597/62736783-385c9700-b9fc-11e9-8c43-6c47e7be933a.gif)
![silence](https://user-images.githubusercontent.com/11216597/62736789-3c88b480-b9fc-11e9-969a-7135f0d71e58.gif)
![altpoison](https://user-images.githubusercontent.com/11216597/62737020-dbadac00-b9fc-11e9-805a-58e955073ba2.gif)

**Comment by ncrecc** [comment]
![altfury](https://user-images.githubusercontent.com/11216597/62781486-1a3b7900-ba86-11e9-85d7-a8ad35bca8d5.gif)
Fury?, with multiuse equipment

**Comment by ncrecc** [comment]
This is being used in speedruns now: https://www.speedrun.com/dicey_dungeons/thread/2ra7n

**Comment by TerryCavanagh** [comment]
Yep: this one's duplicated on the main repo too, but this is a better issue so I'll transfer it.
